dnf5/main.cpp

```diff
@@ -714,6 +724,15 @@ int main(int argc, char * argv[]) try {
 
             context.download_and_run(*context.get_transaction());
         }
+    } catch (libdnf::cli::GoalResolveError & ex) {
+        if (!any_repos_from_system_configuration && base.get_config().installroot().get_value() != "/" &&
```

Comment:
I am not sure whether it is a good place. If I will use `dnf repoquery --installroot <> --releasever 33 something`, I will not get any information`. 

But with `dnf repoquery --installed --installroot <> --releasever 33 something` it would be not nice to get a message.

Ok, it might be not the best place, but good enough - no changes requested